docs/cucumber-messages/index.md

```diff
@@ -0,0 +1,39 @@
+# Reporting
```

Comment:
We should re-think whether we should publicly call this main feature as cucumber messages and not just formatters. For most of the ppl the important bit is the html formatter, they don't really care if we use cucumber messages in the background. Of course cucumber messages can be still explained as one of the supported formatters.
I will also think on this...

Reply:
Are we planning on shipping our own HTML (or other) formatter? Naturally that's what end-users actually care about, I'm just unclear whether we're going to be adding that in, or expecting our end-users to engage with the cucumber messages ecosystem and strap in a tool to progress a more raw output.

Reply:
I've already included the Cucumber/HTMLFormatter that will output HTML files using the Cucumber react components. This results in a static standalone HTML file per test run. 

I agree with the sentiment that we adopt the same language as Cucumber and refer to the overall capability as 'formatters'.

Reply:
Would you like me to rename files and folders from 'Cucumber Messages' to 'Formatters'?

Reply:
Yes, I think that's better.

Reply:
Renamed projects, folders, and namespaces.